backend/typescript/middlewares/validators/campValidators.ts

```diff
@@ -320,3 +320,38 @@ export const updateCampSessionDtoValidator = async (
   }
   return next();
 };
+
+export const editFormQuestionValidator = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  if (req.body.formQuestion && !validateFormQuestion(req.body.formQuestion)) {
+    return res
+      .status(400)
+      .send(getApiValidationError("formQuestion", "string", true));
+  }
+
+  return next();
+};
+
+export const createFormQuestionsValidator = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  if (
+    req.body.formQuestions &&
+    Array.isArray(req.body.formQuestions) &&
+    /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+    !req.body.formQuestions.every((formQuestion: { [key: string]: any }) => {
+      return validateFormQuestion(formQuestion);
+    })
```

Comment:
can we do this instead?
```suggestion
    !req.body.formQuestions.every(validateFormQuestion)
```